crml/nft/src/lib.rs

```diff
@@ -20,22 +20,34 @@
 //!
 //! Intended to be used "as is" by dapps and provide basic NFT feature set for smart contracts
 //! to extend.
+//!
+//! *Collections*:
+//!  A namespacing tool for logical grouping of related tokens
+//!  Tokens within a collection can have the same royalties fees, metadata base URIs, and owner address
+//!
+//! *Series*:
+//! A grouping of tokens within a collection namespace
+//! Tokens in the same series will have exact attributes and royalties.
```

Comment:
this implies that the series has control over the royalties. Maybe
"Tokens in the same series will have exact attributes and inherit it royalties info from the collection."